app/controllers/concerns/kinetic_cafe/error_handler.rb

```diff
@@ -71,13 +71,17 @@ def kinetic_cafe_error_handler(error, error_params = {})
 
     kinetic_cafe_error_log_error(error)
 
-    respond_to do |format|
-      format.html do
-        kinetic_cafe_error_render_html(error)
-      end
-      format.json do
-        kinetic_cafe_error_render_json(error)
+    if respond_to?(:respond_to)
+      respond_to do |format|
+        format.html do
```

Comment:
I tried that, and rubocop failed because we use `do..end` for procedural blocks (Like `each`) and `{  ... }` for blocks that are used with the intention of returning a value (Like `map`).